src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -252,17 +315,27 @@
     // Represents rows from stop_times.txt
     public static final GraphQLObjectType stopTimeType = newObject().name("stopTime")
             .field(MapFetcher.field("trip_id"))
-            .field(MapFetcher.field("stop_id"))
-            .field(MapFetcher.field("stop_sequence", GraphQLInt))
             .field(MapFetcher.field("arrival_time", GraphQLInt))
             .field(MapFetcher.field("departure_time", GraphQLInt))
+            .field(MapFetcher.field("stop_id"))
+            .field(MapFetcher.field("stop_sequence", GraphQLInt))
             .field(MapFetcher.field("stop_headsign"))
-            .field(MapFetcher.field("timepoint", GraphQLInt))
-            .field(MapFetcher.field("drop_off_type", GraphQLInt))
             .field(MapFetcher.field("pickup_type", GraphQLInt))
+            .field(MapFetcher.field("drop_off_type", GraphQLInt))
             .field(MapFetcher.field("continuous_drop_off", GraphQLInt))
             .field(MapFetcher.field("continuous_pickup", GraphQLInt))
             .field(MapFetcher.field("shape_dist_traveled", GraphQLFloat))
+            .field(MapFetcher.field("timepoint", GraphQLInt))
+            // Additional GTFS Flex booking rule fields.
+            .field(MapFetcher.field("pickup_booking_rule_id"))
+            .field(MapFetcher.field("drop_off_booking_rule_id"))
+            // Additional GTFS Flex location groups and locations fields
+            .field(MapFetcher.field("start_pickup_dropoff_window", GraphQLInt))
+            .field(MapFetcher.field("end_pickup_dropoff_window", GraphQLInt))
```

Comment:
Change to regular text. In https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md#stop_timestxt-file-extended, the pickup/dropoff window is specified as time, and in one GTFS Flex file we use there is a time string.

Reply:
All other fields that are TIME in the gtfs spec are Int in this file. There might be a conversion happening?

Reply:
A conversion takes place on loading using `Entity.getTimeField()`. I think it is correct to leave these fields as TIME/Int fields.

Reply:
Thanks for checking.